src/cljdoc/doc_tree.clj

```diff
@@ -83,6 +83,13 @@
   (and (.startsWith (.toLowerCase path) "readme.")
        (supported-file-type path)))
 
+(defn starts-with-any? [s coll]
+  (some #(.startsWith s %) coll))
```

Comment:
Since this function is used just once you could also just inline it into the `changelog?` function. Same reasoning as with `filter-first-path` — reasoning about standard library functions is much easier for newcomers to the code and some repetition is totally fine. 🙂